backend/controllers/orgs.go

```diff
@@ -156,71 +156,47 @@ func AssociateTenantIdToDiggerOrg(c *gin.Context) {
 		c.AbortWithStatus(http.StatusForbidden)
 		return
 	}
+	if claims, ok := token.Claims.(jwt.MapClaims); token.Valid && ok {
+		name := claims["name"]
+		tenantId := claims["tenantId"]
 
-	if token.Valid {
-		if claims, ok := token.Claims.(jwt.MapClaims); ok {
-			if claims.Valid() != nil {
-				slog.Warn("Token's claim is invalid")
-				c.AbortWithStatus(http.StatusForbidden)
-				return
-			}
```

Comment:
In v5, we don't need to validate the claims again after parsing the token. 

`jwt.ParseUnverified`, which is called by `jwt.Parse` under the hood, validates the claims for us already: https://github.com/golang-jwt/jwt/blob/v5.3.0/parser.go#L116-L126